unity-renderer/Assets/Scripts/MainScripts/DCL/CRDTProtocol/CRDTUtils.cs

```diff
@@ -0,0 +1,20 @@
+namespace DCL.CRDT
+{
+    public static class CRDTUtils
+    {
+        public static long KeyFromIds(int entityId, int componentId)
+        {
+            return (long)entityId << 32 | (uint)componentId;
```

Comment:
Could you add comments about the bit structure and what is the objective of these operations?

From what I understood, the MSB 32 bits are for the `entityId` and the last ones are for `componentId` 

| entityId | componentId |
|---|---|
| 00000000000000000000000000000000 | 00000000000000000000000000000000 |
| MSB 32 bits | LSB 32 bits |

Is that correct?

Reply:
And what is the reason for casting `componentId` to `uint`? 🤔

Reply:
1- correct, I will add you comment as a comment :P
2- don't know, IDE was complaining about it so I did the casting

Reply:
Maybe could be better to cast it to `long` to make a `OR` operation between two `longs` instead of a `long` and a `uint`.